tests/test_benchmark.py

```diff
@@ -582,6 +583,31 @@ def output_parser(output):
       return 100.0/float(re.search('Unrolled Single  Precision +([\d\.]+) Mflops', output).group(1))
     self.do_benchmark('linpack_float', open(path_from_root('tests', 'linpack.c')).read(), '''Unrolled Single  Precision''', force_c=True, output_parser=output_parser, shared_args=['-DSP'])
 
+  def test_memcpy_128b(self):
```

Comment:
please disable these by default (`zzz_test_*`, as it looks for `test_`). unless you think we should constantly monitor these?